soccer/Robot.cpp

```diff
@@ -235,6 +236,10 @@ const Geometry2d::Segment OurRobot::kickerBar() const {
     return Segment(pose * L, pose * R);
 }
 
+Geometry2d::Point OurRobot::mouthPos() const {
```

Comment:
Yeah I added it because I thought I was going to use it but ended up not, but figured it'd be good to leave in anyways